docs/getting-started.mdx

```diff
@@ -8,46 +8,59 @@ import { AssetsProvider } from '@site/src/components/assets-context';
 import DownloadCard from '@site/src/components/download-card';
 import DownloadCardStyles from '@site/src/css/download-card.module.css';
 
-
-## Get an Account
-
-<Columns className='text--center getting-started-cards'> 
-  <Column style={{paddingBottom: 20}}>
-    <Card shadow='tl'>
-      <CardHeader>
-        <h3>Hosted zrokNet</h3>
-      </CardHeader>
-      <CardBody>
-        Use NetFoundry's public zrok instance.
-      </CardBody>
-      <CardFooter>
-        <a href="https://myzrok.io/">
-        <button className='button button--secondary button--block'>Get an Account</button>
-        </a>
-      </CardFooter>
-    </Card>
-  </Column>
-  <Column style={{paddingBottom: 20}}>
-    <Card shadow='tl'>
-      <CardHeader>
-        <h3>Self-Hosted zrok</h3>
-      </CardHeader>
-      <CardBody>
-        Run a zrok instance on Linux, Docker, or Kubernetes.
-      </CardBody>
-      <CardFooter>
-        <a href="/docs/category/self-hosting/">
-        <button className='button button--secondary button--block'>Guides</button>
-        </a>
-      </CardFooter>
-    </Card>
-  </Column>
+## Secure resource sharing made easy
+
+<Columns className='text--center getting-started-cards'>
+    <Column style={{paddingBottom: 20}}>
+        <Card shadow='tl'>
+            <CardHeader>
+                <h3>Hosted zrok</h3>
+            </CardHeader>
+            <CardBody>
+                Use NetFoundry's public zrok instance.
+            </CardBody>
+            <CardFooter>
+                <a href="https://myzrok.io/">
+                    <button className='button button--secondary button--block'>Get an Account</button>
+                </a>
+            </CardFooter>
+        </Card>
+    </Column>
+    <Column style={{paddingBottom: 20}}>
+        <Card shadow='tl'>
+            <CardHeader>
+                <h3>Self-Hosted zrok</h3>
+            </CardHeader>
+            <CardBody>
+                Run a zrok instance on Linux, Docker, or Kubernetes.
+            </CardBody>
+            <CardFooter>
+                <a href="/docs/category/self-hosting/">
+                    <button className='button button--secondary button--block'>Guides</button>
+                </a>
+            </CardFooter>
+        </Card>
+    </Column>
 </Columns>
 
+* Sign up for a hosted zrok account in the link above to enable your account token
+* Download the zrok binary for your platform [here](#installing-the-zrok-command)
+* Run `zrok enable <your_account_token>` to enable your shell environment for sharing
```

Comment:
```suggestion
* Run `zrok enable <your_account_token>` to activate your account in your user environment.
```

Maybe we shouldn't emphasize shell env since we're not technically setting any shell env vars, and we have GUI user experiences around the corner that will make managing zrok shares less terminal-centric.